deploy/cht_sync/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
I'm pretty sure this file already exists in the repo, so I'm unsure why this PR reports that it's new. I wonder if you could rebase off of `main` so we only end up seeing relevant changes. That would be super helpful.